composer.json

```diff
@@ -17,16 +17,16 @@
 	],
 	"require": {
 		"php": ">=5.3.2",
-		"symfony/form": "~2.3",
-		"symfony/http-kernel": "~2.3",
-		"symfony/translation": "~2.3",
-		"symfony/yaml": "~2.3"
+		"symfony/form": "^2.6.4|~3.0",
+		"symfony/http-kernel": "^2.6.4|~3.0",
+		"symfony/translation": "^2.6.4|~3.0",
+		"symfony/yaml": "^2.6.4|~3.0"
```

Comment:
I've already started upgrading the project, but encountered some issues. However, I want to use some of the new features of the bundle. Let's see how things turn out. I could still bump to Symfony 2.7 before releasing 3.0. This doesn't need to happen in this PR. :smirk: